src/PimcoreMonitorBundle/Command/HealthReportCommand.php

```diff
@@ -118,8 +118,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             RedirectionExceptionInterface |
             ServerExceptionInterface $e
         ) {
+            $jsonResponse = json_decode($response->getContent(false), true);
```

Comment:
```suggestion
            $jsonResponse = \json_decode($response->getContent(false), true);
```

Symfony Standard, see https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/3048